pkg/server/pagination/pagination.go

```diff
@@ -31,7 +31,7 @@ const (
 // execution is responded to using the passed in gin.Context. The return value corresponds to whether there was an
 // error within the function.
 func ParsePaginationParams(c *gin.Context, pageRequest *PageRequest) bool {
```

Comment:
rename to `ParsePaginationQueryValues`?

Reply:
Done in [80a4239](https://github.com/TBD54566975/ssi-service/pull/627/commits/80a42397a33b11e0e6487fe0a2e485b6d3323eac)